cantabular-import/minio/IMPORTANT.md

```diff
@@ -0,0 +1,8 @@
+### IMPORTANT! ###
+
+## Do not delete the `data` directory! This directory has special permissions ##
+## properties to allow minio to read and write to it. If you do need to       ##
+## recreate it, make sure to change ownership to user 1001:1001               ##
+## `sudo chown -R 1001:1001 data`                                             ##
+## Note that these permissions settings mean you will need to use super-user  ##
+## privileges to directly edit files in this directory                        ##
```

Comment:
The above comment does not look quite right when viewed in its md format. I would suggest putting some of the lines together to make it more readable e.g. 
```suggestion
## Do not delete the `data` directory! This directory has special permissions properties to allow minio to read and write to it. 
## If you do need to recreate it, make sure to change ownership to user 1001:1001               ##
## `sudo chown -R 1001:1001 data`                                             ##
## Note that these permissions settings mean you will need to use super-user privileges to directly edit files in this directory                      
```